capa/features/extractors/viv/basicblock.py

```diff
@@ -92,22 +93,27 @@ def is_mov_imm_to_stack(instr: envi.archs.i386.disasm.i386Opcode) -> bool:
     if not src.isImmed():
         return False
 
-    if not isinstance(dst, envi.archs.i386.disasm.i386SibOper) and not isinstance(
-        dst, envi.archs.i386.disasm.i386RegMemOper
+    if (
+        not isinstance(dst, envi.archs.i386.disasm.i386SibOper)
+        and not isinstance(dst, envi.archs.i386.disasm.i386RegMemOper)
+        and not isinstance(dst, envi.archs.arm.disasm.ArmRegOper)
     ):
         return False
 
     if not dst.reg:
         return False
 
-    rname = dst._dis_regctx.getRegisterName(dst.reg)
-    if rname not in ["ebp", "rbp", "esp", "rsp"]:
+    if isinstance(dst, (envi.archs.i386.disasm.i386SibOper, envi.archs.i386.disasm.i386RegMemOper)):
+        rname = dst._dis_regctx.getRegisterName(dst.reg)
+    else:
+        rname = dst.reg
+    if rname not in ["ebp", "rbp", "esp", "rsp", envi.archs.arm.disasm.REG_SP, envi.archs.arm.disasm.REG_BP]:
```

Comment:
can this be simplified to just use the constant or the name?

Reply:
So we import envi.archs.arm.disasm.REG_SP as REG_SP and just use REG_SP for example ?

Reply:
yes, exactly

Reply:
@mr-tz i'm not sure i agree, because then it won't be obvious to a reader if the SP/BP are related to arm vs x86 vs ...

Reply:
I meant to use:
- envi.archs.i386.disasm.REG_EBP (or similar [untested])
- envi.archs.amd64.disasm.REG_RBP (or similar [untested])
- etc.

Instead of "ebp", "rbp", etc.